lib/stove/cli.rb

```diff
@@ -62,10 +60,11 @@ def execute!
       # If we got this far, everything was successful :)
       @kernel.exit(0)
     rescue => e
-      log.error('Stove experienced an error!')
-      log.error(e.class.name)
-      log.error(e.message)
-      log.error(e.backtrace.join("\n"))
+      Stove::Log.init($stderr)
```

Comment:
@tas50 @tyler-ball The problem wasn't in the cucumber tests, but the actual code to log
the error to `$stderr`. This line fixed everything. 🥇 

![tenor-112058542](https://user-images.githubusercontent.com/5712253/73406149-f7f71b00-42b2-11ea-8b1e-758ebf678293.gif)